package.json

```diff
@@ -160,7 +160,7 @@
 
                 "configurationSnippets": [
                     {
-                        "label": "PowerShell: Launch Current File Configuration",
+                        "label": "PowerShell: Launch Current Script",
```

Comment:
Script is better than file but you might want to leave in "Configuration" at the end.  This is what I see for the other langs that support debug configuration snippets:
![image](https://cloud.githubusercontent.com/assets/5177512/21793582/d6cd4386-d6af-11e6-8b1c-2bbed732ab72.png)